File: Tamagotchi_ApiTests/shop/databuilder/Food_Factory.java
package com.tamagotchi.databuilder;

import com.tamagotchi.shop.model.Food;
import com.tamagotchi.shop.model.ModelFood;

public class Food_Factory {
	
	
	public Food GenerateFood() {
		Food food = new Food();
		food.id = "5a5f49cf-0618-4052-9fb9-6441fe9dea3b";
		food.name = "whinegums";
		food.category = "candy";
		food.price = 1.70;
		food.discount = 0;
		food.energyVal = 4;
		food.happyVal = 1;
		food.foodVal = 8;
		
		return food;
	}
}

File: Tamago_WebApi's/Inventory_Test/Factory/Food_Factory.cs
using Shared.Shared.ApiModels;
using System;
using System.Collections.Generic;
using System.Text;

namespace Inventory_Test.Factory
{
	public static class Food_Factory
	{
		public static FoodModel GenerateFood(int seed)
		{
			int maxvalue = 10;

			var r = new Random(seed);
			FoodModel food = new FoodModel();

			food.id = Guid.NewGuid();
			food.name = "test-gummies";
			food.category = "candy";
			food.price = r.Next(0, maxvalue) + r.NextDouble();
			food.discount = 0.0;
			food.energyVal = r.Next(0, maxvalue);
			food.foodVal = r.Next(0, maxvalue);
			food.happyVal = r.Next(0, maxvalue);

			return food;
		}

		public static List<FoodModel> GenerateFood(int amount, int seed)
		{
			var foodList = new List<FoodModel>();

			for (int i = 0; i < amount; i++)
				foodList.Add(GenerateFood(seed));

			return foodList;
		}
	}
}
